strider/server.py

```diff
@@ -357,10 +357,16 @@ async def multi_lookup(callback, queries: dict, query_keys: list, redis_client:
 
     async def single_lookup(query_key):
         query_result = await lookup(queries[query_key], redis_client)
-        async with httpx.AsyncClient(timeout=httpx.Timeout(timeout=600.0)) as client:
-            await client.post(callback, json=query_result)
-
-    await asyncio.gather(*map(single_lookup, query_keys))
+        try:
+            async with httpx.AsyncClient(
+                timeout=httpx.Timeout(timeout=600.0)
+            ) as client:
+                await client.post(callback, json=query_result)
+        except Exception as e:
+            LOGGER.debug(e)
+            raise e
```

Comment:
Do we want to raise an error here? Or just continue? How do we want to handle failed callbacks?

Reply:
Considering how we'd talked about using the `return_exceptions=True` flag in the gather on line 369, I thought raising it made the most sense. But maybe we just want to throw it in the logs and continue. I'm open to either.